### DISCUSSION TITLE
Email when your jobs are finished

### DISCUSSION THREAD

**Discussion by stewarthe6** [llnl/maestrowf#327] [Ideas]
It'd be nice if you got an email when your jobs finished or crashed. Maybe add a flag for an email when you start the run and once your conductor exits it sends an email you with status.csv in it or something.

**Comment by FrankD412** [comment]
I think a command line option would be great for this. Something like: `maestro run --notify <email> ...`; Emailing the status file might be a bit much, as I know some users who run rather large studies and so trying to html format that in an email or something might be painful. What sort of summary information might you want to see in that email?

**Reply by stewarthe6** [reply]
Yeah, the status gets kind of hard to look at kind of fast. I have a little script that looks at status.csv and counts some of the states that I care about. It looks something like this:
```
finished
409
failed
0
timedout
0
running
90
pending
10
```
It probably wouldn't need pending or running. At minimum, it just needs to let you know that maestro is shutting down.
